src/components/Board/BoardProvider.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+const BorderContext = React.createContext()
+
+export default class BoardProvider extends React.Component {
```

Comment:
I've never seen this pattern before. In a sentence, what is going on here?

Reply:
@nickeyvee I'm going to use for the Modal (my next PR). For now, it doesn't do anything.